app/views/errors/show.html.haml

```diff
@@ -17,7 +17,11 @@
               %span.badge.badge-environment= @error.environment
             %li
               %span.label.badge= @error.checksum
-        .col-xs-4.text-right
+        .col-xs-5.text-right
+          - github = @website.integrations.find{ |integration| integration.provider == 'github' }
+          - if github
```

Comment:
Ugh so ugly. Why don't you make a method in website called `github_integration?` and then check here with `if @website.github_integration?`

You can make these methods using this code so you don't write a method for each integration:
```
  Integrations.drivers_types.each do |integration|
    class_eval <<-EOV, __FILE__, __LINE__
      def "#{integration.to_s}_integration?"
        self.integrations.find_by(provider: integration)
      end
    EOV
  end
```